pkgs/development/libraries/science/math/openblas/default.nix

```
@@ -76,6 +76,13 @@ let
       USE_OPENMP = true;
     };
 
+    loongarch64-linux = {
+      BINARY = 64;
+      TARGET = setTarget "LOONGSONGENERIC";
+      DYNAMIC_ARCH = setDynamicArch true;
```

```suggestion
      DYNAMIC_ARCH = setDynamicArch false;
```

And this fixes the build, finally.